### ISSUE TITLE
empty symbol

### ISSUE THREAD

**Issue by hyzgh** [jung-kurt/gofpdf#294]
I use MultiCell to generate a pdf, it works well when there is not missing symbols. But when there are some missing symbols, it will generate a text with some empty, please see the picture in the following.
```go
pdf := gofpdf.New("P", "mm", "A4", "font/")
pdf.AddUTF8Font("stsong", "", "stsong.ttf")
pdf.SetFont("stsong", "", 14)
pdf.AddPage()
txtStr, err := ioutil.ReadFile(fileName)

pdf.MultiCell(0, 7, string(txtStr), "", "", false)
if pdf.Err() {
    fmt.Println(pdf.Error())
    continue
}
pdf.Ln(-1)

err = pdf.OutputFileAndClose(fileName + ".pdf")
if err != nil {
    panic(err)
}
```
txt: 
![image](https://user-images.githubusercontent.com/31174102/63584905-806ce380-c5d0-11e9-9f72-def63dd105c0.png)

pdf: 
![image](https://user-images.githubusercontent.com/31174102/63584974-a72b1a00-c5d0-11e9-901b-9618261dabf9.png)

I would like to know whether there is a way to know the text will be empty when generating the pdf.

**Comment by jung-kurt** [comment]
Thanks for report, @hyzgh. Can you provide the text file referenced by `filename` as a file rather than a PNG image? This will assist in solving this.

**Comment by hyzgh** [comment]
Here it is. [10.txt](https://github.com/jung-kurt/gofpdf/files/3534278/10.txt)

**Comment by ArtemKor** [comment]
Are you sure the font stsong.ttf contains malagasy glyphs?

**Comment by hyzgh** [comment]
@ArtemKor malagasy glyphs?

**Comment by ArtemKor** [comment]
@hyzgh Sorry, I made a mistake in defining the alphabet. But the question remains the same: Are you sure the font stsong.ttf contains Tibetan glyphs?

**Comment by hyzgh** [comment]
@ArtemKor No, it doesn't. I need to transfer some `.txt` files to  `.pdf` files, but it seems that I could only check `.pdf` on my own to confirm whether there is some failing glyphs. And I would like to know whether there is a way to know the text will be empty by the code.
